util/cron/nightly

```diff
@@ -710,6 +714,15 @@ if ($runtests == 0) {
     }
 
     $ENV{'CHPL_HOME'} = $chplhomedir;
+
+    # Confirm Chapel built correctly before start_test / paratest
+    if ($buildcheck eq 1) {
```

Comment:
I *think* numerical comparison in perl is done with `==` and `eq` is from string comparison. (I'd double check me on that though)